backend/audit/validators.py

```diff
@@ -250,6 +250,8 @@ def validate_use_of_gsa_migration_keyword(general_information, is_data_migration
         general_information.get("auditee_uei", ""),
         general_information.get("ein", ""),
         general_information.get("auditor_ein", ""),
+        general_information.get("is_sp_framework_required", ""),
```

Comment:
`is_low_risk_auditee` and `is_sp_framework_required` are not part of general_information section. They are part of AuditInformation

Reply:
Created new validate function for checking gsa_migration in audit_information.